### DISCUSSION TITLE
Diggin' this terminal

### DISCUSSION THREAD

**Discussion by mard0** [mkrueger/icy_term#32] [Ideas]
It's looking really good so far, Have been using it the last few days since someone dropped a link for it on fsxnet. Heres some requests that I think would be nice.

1. Add a hotkey to toggle/disable the scrollbar and the upper toolbar.
2. add alt-enter as an option for fullscreen.
3: add ICE !!

thanks

**Comment by mkrueger** [comment]
Added F11 hotkey for fullscreen support - that disables the upper toolbar as well. 
(it's possbile to reconfigure to alt+enter in the options)

That ICE doesn't work was a bug - should be fixed. 

Grab 0.6.2 or newer.
